docs/index.rst

```diff
@@ -0,0 +1,9 @@
+Pyhound pdf
+===========
+
+
+This is the pdf harvester for pyhound.
+
+See the `[README]`_ for more information & examples.
+
+.. _[README]: https://github.com/Polyconseil/pyhound
```

Comment:
What is the point of having a "docs" folder if it has less information than the README file? I would think that the README file is enough for now.